apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx

```diff
@@ -1,19 +1,47 @@
 import React, {PropTypes} from 'react';
-import {Button, FormControl} from 'react-bootstrap';
-import {Facilitator1819Program} from './detail_view_facilitator_specific_components';
+import {Button, FormControl, Panel} from 'react-bootstrap';
+import Facilitator1819Questions from './detail_view_facilitator_specific_components';
 import $ from 'jquery';
+import _ from 'lodash';
+
+// Public function to render a question/answer pair as either a line item or a panel
+// depending on whether the question will be assigned score
+const renderItem = (key, value, layout, iteratorKey) => {
+  if (typeof value === 'string' || typeof value === 'object') {
+    let renderedValue = value;
+    if (Array.isArray(value)) {
+      renderedValue = _.join(value, ', ');
+    }
 
-const renderLineItem = (key, value) => {
+    if (layout === 'lineItem') {
+      return renderLineItem(key, renderedValue, iteratorKey);
+    } else {
+      return renderQuestionBox(key, renderedValue, iteratorKey);
+    }
+  }
+};
```

Comment:
Why export a function like this instead of a component (even a stateless functional component)?

Reply:
True - I can make this a component easily enough